Core/DISET/private/Transports/SSL/SocketInfoFactory.py

```diff
@@ -54,7 +55,11 @@ def __socketConnect( self, hostAddress, timeout, retries = 2 ):
     return S_ERROR( ", ".join( errs ) )
 
   def __sockConnect( self, hostAddress, sockType, timeout, retries ):
-    osSocket = socket.socket( sockType, socket.SOCK_STREAM )
+    try:
+      osSocket = socket.socket( sockType, socket.SOCK_STREAM )
+    except Exception as e:
+      gLogger.warn( "ipv6 can not be used! Rollback to ipv4! Please update your machine", e ) 
```

Comment:
The warning message is irrelevant, exception can be not due to ipv6 connection but anything else. The message can be just "Exception while creating a socket". 

The exception thrown is of socket.error type. 

When casting an exception to string (in the log message) it is better to do it explicitly, i.e. str( e )